test/test_integration_single.rb

```diff
@@ -71,28 +73,28 @@ def test_term_not_accepts_new_connections
   def test_int_signal_with_background_thread_in_jruby
```

Comment:
This method name could be updated to `test_int_shutdown` or `test_int_exit` while this file is being modified.
Also, `skip_unless :jruby` can be removed.

Reply:
Done, but not yet pushed.